modules/pulp/templates/pulp_sign.sh.epp

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+export GNUPGHOME=/home/pulp/.gnupg
+
+FILE_PATH=$1
+SIGNATURE_PATH="$1.asc"
+
+PUBLIC_KEY_PATH="$(cd "$(dirname $1)" && pwd)/public.key"
+
+ADMIN_ID="$gpg_key_id"
```

Comment:
The unescaped dollars give me slight pause. To make the difference between template and shell variables more explicit we could turn this into an ERB file and use:

```suggestion
ADMIN_ID="<%= gpg_key_id %>"
```

That might need to be `@gpg_key_id`, I need to refresh how puppet sets up erb rendering.